.github/workflows/ci.yml

```
@@ -0,0 +1,33 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    branches:
+      - development
+      - main
+
+jobs:
+  lint:
+    name: autoline build test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v2
+
+      - name: Install NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install Dependencies
+        run: |
+          yarn install
```

Please use the dependencies install cache. It will save us hours

Here's the action which I used initially - https://github.com/bahmutov/npm-install

```suggestion
      - name: Install Dependencies
         uses: bahmutov/npm-install@v1.6.0
         with:
           useRollingCache: true
```